<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gis Client application</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="PwGisWidget" name="gisWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>300</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Button">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Base">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
          <colorrole role="Window">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>85</red>
             <green>170</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 170, 255);</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="Measure">
        <property name="text">
         <string>Measure: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="measureData">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbMouseMoveEvents">
        <property name="text">
         <string>Show mouse move</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="testEditFeatureButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>randomize path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="testEditFeatureButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>add Points to Path</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="testEditFeatureButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>remove Path Points</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="testFormats">
        <property name="text">
         <string>test Supported Formats</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnInit">
        <property name="text">
         <string>Init map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnKmlWorldMap">
        <property name="text">
         <string>Load Kml World map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestMapBounds">
        <property name="text">
         <string>Map Bounds Check</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestMapFormats">
        <property name="text">
         <string>Map Formats</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReload">
        <property name="text">
         <string>Reload map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTestCaptionAndTooltip">
        <property name="text">
         <string>Test Caption and Tooltip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDemoUpdateFeatureParameters">
        <property name="text">
         <string>Demo Update Feature Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="removeGradFeature">
        <property name="text">
         <string>Удалить Сектор с Градиентом</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addGradFeature">
        <property name="text">
         <string>Добавить Сектор с Градиентом</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetMarker">
        <property name="text">
         <string>Set Marker</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeleteMarker">
        <property name="text">
         <string>Delete Marker</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnTransactedAdd">
        <property name="text">
         <string>Transactions Benchmark</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QPushButton" name="btnGetFeatureProperties">
        <property name="text">
         <string>Get Feature Properties</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSetCustomStyle">
        <property name="text">
         <string>Set Custom Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCustomStyleMarker">
        <property name="text">
         <string>Custom Style Marker</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveAllObjects">
        <property name="text">
         <string>Clear map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bntInvalidMap">
        <property name="text">
         <string>Invalid map</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>LoadProgress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="btnTooltipsAndCaptions">
        <property name="text">
         <string>Apply Style</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="styleEdit">
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QPushButton" name="btnDrawCircle">
        <property name="text">
         <string>Draw Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCenterLat89">
        <property name="text">
         <string>Center Lat=89</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCenterLonM179">
        <property name="text">
         <string>Center Lon=-179</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddSlice">
        <property name="text">
         <string>Add Slice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveSlice">
        <property name="text">
         <string>Remove Slice</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd2Markers">
        <property name="text">
         <string>Add 2 Markers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove2Markers">
        <property name="text">
         <string>Remove 2 Markers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddMarker">
        <property name="text">
         <string>Add Marker</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnUpdateMarker">
        <property name="text">
         <string>Update Marker</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd4Objects">
        <property name="text">
         <string>Add 4 Objects</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMove4Objects">
        <property name="text">
         <string>Move 4 Objects</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QPushButton" name="btnDrawLongLine">
        <property name="text">
         <string>Draw Long Line</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDrawLongRectangle">
        <property name="text">
         <string>Draw Long Rectangle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDrawBrokenCircle">
        <property name="text">
         <string>Draw Broken Circle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddForkedObjects">
        <property name="text">
         <string>Add Forked Objects</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemoveForkedObjects">
        <property name="text">
         <string>Remove Forked Objects</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1339</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PwGisWidget</class>
   <extends>QWidget</extends>
   <header location="global">PwGis/pwgiswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
